plugins/omemo/src/trust_manager.vala

```diff
@@ -247,6 +247,7 @@ public class TrustManager {
             StanzaNode? _encrypted = stanza.stanza.get_subnode("encrypted", NS_URI);
             if (_encrypted == null || MessageFlag.get_flag(stanza) != null || stanza.from == null) return false;
             StanzaNode encrypted = (!)_encrypted;
+            if (message.body == null) message.body = "[This message is OMEMO encrypted]";
```

Comment:
Can't this hard-coded string be generalized with the other string that Dino *sets* for outgoing OMEMO messages using a global? Or using i14n if that's already available.